lib/LANraragi/Controller/Api/Readinggroup.pm

```diff
@@ -0,0 +1,107 @@
+package LANraragi::Controller::Api::Readinggroup;
```

Comment:
No, no, I just named it that way because it's the first thing that came to my mind in that moment(And ended up regretting it due to naming conventions of Mojo). If you tell me which you think would be the adequate name, then I'll rename all the modules.

Just in the topic, while Tanks would be the main use-case for this module, it's important to mention that right now it's possible to create lists for generic groups(Like "Pending to read") where the groups might not be of "Series" related works, but "Reader" related works, which also means a file could be included in more than one list, but if this sounds too complex, I can make it so an arcid can only be in one list at a time. Here I'll let you decide which sounds better, and what name would work the best for the module(In my opinion, Tankoubon sounds good).

Reply:
Let's go with Tankoubon then 👍 

My thought is that stuff like "pending to read" is more suited for categories, but considering how similar the two concepts are I wouldn't be surprised if people made that with tanks instead so they could reorder them.  

I don't think we should restrict arcid from being in more than one tank, that'd be a virtual limitation more than anything else 🤔